lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb

```diff
@@ -68,12 +73,23 @@ def generate
           # that use Swift. Setting this for the embedded target would
           # cause an App Store rejection because frameworks cannot be embedded
           # in embedded targets.
-          if !target.requires_host_target? && pod_targets.any?(&:uses_swift?)
-            config['EMBEDDED_CONTENT_CONTAINS_SWIFT'] = 'YES'
-            config['ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES'] = 'YES'
-          else
-            config['ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES'] = 'NO'
+
+	  # Extract swift_version number
+	  @swift_version = target.resolved_build_setting('SWIFT_VERSION').values.compact.uniq
+	  embed_default_value = 'NO'
+
+	  # Check if swift_version is prior to SWIFT_VERSION_CHECK
+	  if !target.requires_host_target? && pod_targets.any?(&:uses_swift?)
+	    embed_default_value = 'YES'
+	    if @swift_version < SWIFT_VERSION_CHECK
```

Comment:
Couple of points here:
1. This is doing a String comparison which would break if they eventually add patch versions into the mix due to being a lexicographic compare.

2. I think this should be `<=` to also use these keys on Xcode 8 Swift 2.3 projects